Decred Wallet/Extensions/UIButton.swift

```diff
@@ -54,4 +53,22 @@ extension UIButton {
             self.setNeedsLayout()
         }
     }
+
+    private func imageWithColor(color: UIColor) -> UIImage? {
```

Comment:
- Is this `imageWithColor` function in any way related to `UIButton`?
- The function name isn't really quite clear to me. What does `imageWithColor` do?